lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -826,8 +826,8 @@ def stop_job(self, job_wrapper):
             job_to_delete = find_job_object_by_name(self._pykube_api, job.get_job_runner_external_id(), self.runner_params['k8s_namespace'])
             if job_to_delete and len(job_to_delete.response['items']) > 0:
                 k8s_job = Job(self._pykube_api, job_to_delete.response['items'][0])
-                if job_wrapper.guest_ports:
-                    self.__cleanup_k8s_interactivetools(job_wrapper, k8s_job)
+                if self.__has_guest_ports(job_wrapper.get_state()):
```

Comment:
That's the state column of the job. Maybe refactor `__has_guest_ports` to take the job_wrapper directly ?

Reply:
Thanks for pointing that out @mvdbeek. Have fixed it now.